packages/module-web/src/client/core/schema/components/page/Page.tsx

```
@@ -117,15 +119,22 @@ const InternalPage: React.FC = (props) => {
-          </div>
+          <>
+            {Object.values(extendComponents)?.map((item: any) => {
+              const schema = {
+                type: 'void',
+                name: item.name,
+                'x-component': item.component,
+                'x-comonent-props': {},
+              };
+              const componentProps = { ...props, isHeaderEnabled, fieldSchema, enabledSharePage };
+              return (
+                <PageExtendComponentProvider {...componentProps}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)